setup.py

```diff
@@ -36,6 +36,7 @@
         "c42eventextractor==0.4.0",
```

Comment:
I believe we also need to add `!=3.5.*` to `python_requires=`

Reply:
Updated.